extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md

```diff
@@ -0,0 +1,125 @@
+
+To use this binding, you first need to [register and get your API key](https://www.wunderground.com/weather/api/d/pricing.html) .
+
+## Supported Things
+
+There is exactly one supported thing type, which represents the weather information for an observation location. It has the id `weather`.
+
+## Discovery
+
+There is no discovery implemented. You have to create your things manually.
+
+## Binding Configuration
+ 
+The binding has no configuration options, all configuration is done at Thing level.
+ 
+## Thing Configuration
+
+The thing has a few configuration parameters:
+
+| Parameter | Description                                                              |
+|-----------|------------------------------------------------------------------------- |
+| apikey    | API key to access the Weather Underground service. Mandatory.            |
+| location  | Location to be considered by the Weather Underground service. Mandatory. |
+| language  | Language to be considered by the Weather Underground service. Optional, the default value is EN (English). |
+| refresh   | Refresh interval in minutes. Optional, the default value is 30 minutes.  |
+
+For the location parameter, different syntaxes are possible:
+
+| Syntax                  | Example          |
+|-------------------------|----------------- |
+| US state/city           | CA/San_Francisco |
+| US zipcode              | 60290            |
+| country/city            | Australia/Sydney |
+| latitude,longitude      | 37.8,-122.4      |
+| airport code            | KJFK             |
+| PWS id                  | pws:KCASANFR70   |
+
+
+## Channels
+
+The weather information that is retrieved is available as these channels:
+
+| Channel Group ID | Channel ID | Item Type    | Description              |
+|------------------|------------|--------------|------------------------- |
+| Current | location | String | Weather observation location |
+| Current | stationId | String | Weather station identifier |
+| Current | observationTime | DateTime | Observation date and time |
+| Current | conditions | String | Weather conditions |
+| Current | temperatureC | Number | Temperature in degrees Celsius |
+| Current | temperatureF | Number | Temperature in degrees Fahrenheit |
+| Current | relativeHumidity | Number | Relative humidity in % |
+| Current | windDirection | String | Wind direction |
+| Current | windDirectionDegrees | Number | Wind direction in degrees |
+| Current | windSpeedKmh | Number | Wind speed in km/h |
```

Comment:
But the service is providing the values in several units, I would like to keep an access to the provider raw value rather than than considering a conversion done by ESH.
But when conversion unit feature is implemented in ESH, I will change "bonus" channels into advanced channels.